ion-schema/src/isl/mod.rs

```diff
@@ -555,6 +553,7 @@ impl<'a> Iterator for SchemaIterator<'a> {
 }
 
 #[cfg(test)]
+#[allow(unused_must_use)]
```

Comment:
🗺️ The combination of `rstest` and a test function that returns `Result` causes a warning for an unused `Result`. Unfortunately, trying to add this `allow` annotation to the test functions directly did not work (probably because it doesn't get picked up by rstest's macro), so I've applied it to this entire module.